app/models/merchant.rb

```diff
@@ -18,10 +18,13 @@ def top_5_customers
 
   def not_shipped_items
     # require 'pry'; binding.pry
-    Item.joins(invoices: [:invoice_items])
-    .select("items.*, invoices.created_at as creation_date, invoice_items.status")
+    # Item.joins(invoices: [:invoice_items])
```

Comment:
If you don't need this, lets go ahead and remove it so we don't have a cluttered codebase.